src/main/scala/is/hail/annotations/aggregators/RegionValueMaxIntAggregator.scala

```diff
@@ -7,15 +7,18 @@ class RegionValueMaxBooleanAggregator extends RegionValueAggregator {
   private var found: Boolean = false
 
   def seqOp(i: Boolean, missing: Boolean) {
-    found = true
-    if (i > max)
+    if (!missing && (i > max || !found)) {
```

Comment:
I think it's clearer to use `empty` instead of `found` since the check is mostly `!found`. Secondarily, this can only help performance in the `seqOp`. Feel free to push back.